telegram/_videochat.py

```diff
@@ -163,7 +178,9 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["VideoChatSch
 
         data["start_date"] = from_timestamp(data["start_date"])
 
-        return cls(**data, bot=bot)
+        obj = cls(**data)
```

Comment:
why not call super()

Reply:
because my search-&-replace didn't cover that 😄